### PR TITLE
View for landing

### PR DESCRIPTION
**Pull request by IleJok**
Landing page for "AW" renders active positions you've added. On the top of the list is the recent addition. If you don't have active positions, the page renders with some instructions on how to use the site.

### CODE REVIEW

File: front/src/indexApp.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 import React from "react";
 import ReactDOM from "react-dom";
 import App from './App'
 import * as serviceWorker from "./serviceWorker";
 
 
 ReactDOM.render(
-  <App/>,
+  <Router history={history}>
+  <App/></Router>,
```
Comment (rrudd) [inline front/src/indexApp.js]: ?

File: front/src/Containers/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  async componentDidMount() {
+    let userId = sessionStorage.getItem('userId');
+    await GetPositionsByUser(userId, response => {
+      if (response.status === 200) {
+        let allPositions = response.data;
+        this.setState({
+          positions: allPositions,
+          filteredPositions: allPositions,
+          noServerResponse: false,
```
[the remark this answers is not in the document]
Comment (IleJok) [reply front/src/Containers/LandingPage.js]: removed

File: front/src/Containers/PositionFilterForAm.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                        status={position.positionStatus}
+                        skills={position.positionSkills}
+                    />
+                );
+            } else {
+                return "";
+            }
+        });
+        return activePositions.length > 0 ? (
+            <Container>{activePositions} </Container>
```
[the remark this answers is not in the document]
Comment (IleJok) [reply front/src/Containers/PositionFilterForAm.js]: Changed the logic here, does it look better?

File: front/src/Containers/PositionSelection.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ class PositionSelection extends Component {
         positionSkills: [],
         isActive: true
       },
-      userIsConsultant: true,
+      userIsConsultant: "",
```
[the remark this answers is not in the document]
Comment (IleJok) [reply front/src/Containers/PositionSelection.js]: Changed the variable name

**Pull request merged**